template/.github/workflows/preview.yml

```
@@ -0,0 +1,27 @@
+name: Netlify Preview
+on:
+  push:
+    branches-ignore: ['main', 'master']
+jobs:
+  build:
```

Should this be `deploy-preview` job?
```suggestion
  deploy-preview:
```

---

✅  ab79900